posts/tests.py

```diff
@@ -1,3 +1,312 @@
-# from django.test import TestCase
+import pytest
+from .models import Poll, PollFile, Choice
+from django.contrib.auth.models import User
 
-# Create your tests here.
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+DESCRIPTION = "this is a test poll"
+FILE1 = "Alon_Shakaroffs_resume.pdf"
+FILE2 = "Olive.png"
+CHOICETEXT1 = "First option"
+CHOICETEXT2 = "Second option"
+
+
+@pytest.fixture
+def new_user():
+    return User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+
+
+@pytest.fixture
+def new_poll(new_user):
+    return Poll(author=new_user, description=DESCRIPTION)
+
+
+@pytest.fixture
+def new_pollFile(new_poll):
+    return PollFile(poll=new_poll, file=FILE1)
+
+
+@pytest.fixture
+def new_choice(new_poll):
+    return Choice(poll=new_poll, choice_text=CHOICETEXT1)
+
+
+# -------------------------------------------- Poll tests --------------------------------------------
+
+
+# Check that the Poll values are the same as the Poll inputs.
+@pytest.mark.django_db()
+def test_new_poll(new_poll):
+    assert new_poll.author.username == USERNAME
+    assert new_poll.author.first_name == FIRSTNAME
+    assert new_poll.author.last_name == LASTNAME
+    assert new_poll.author.password == PASSWORD
+    assert new_poll.author.email == EMAIL
+    assert new_poll.description == DESCRIPTION
+    assert new_poll.amount_of_voters == 0
+    assert len(new_poll.pollfile_set.all()) == 0
+    assert len(new_poll.choice_set.all()) == 0
+    new_poll.author.save()
+    assert new_poll.author.profile.profile_pic == 'profile-icon.png'
+
+
+# Check if the Poll is saved in the database and accessible via his author.
```

Comment:
Same here, change it to : "...via it's author." instead of "...via his author.".